docs/changelog-fragments/576.bugfix.rst

```diff
@@ -0,0 +1,7 @@
+pylibssh no longer crashes when received EOF or when channel is not explicitly
+closed -- by :user:`pbrezina`.
+
+Previously, pylibssh crashed if ``channel.recv`` was called and ``libssh``
```

Comment:
I just realized that this refers to `pylibssh` which is not an identifier used anywhere in this project. The PyPI project is `ansible-pylibssh` and the corresponding GitHub slug is `ansible/pylibssh`. The importable is `pylibsshext`. Let's try using the project name here. I hope the `|project|` RST substitution declared in `docs/conf.py` will just work:

```suggestion
|project| no longer crashes when received EOF or when channel is not explicitly
closed -- by :user:`pbrezina`.

Previously, |project| crashed if ``channel.recv`` was called and ``libssh``
```